src/reducers/Users.js

```diff
@@ -0,0 +1,19 @@
+import { UserConstants } from '../actions/User'
+
+export default function Users(state = [], action = {}) {
+  switch (action.type) {
+
+    case UserConstants.FETCH_ERROR:
+      return state
+
+    case UserConstants.FETCH_SUCCESS:
+      const { user } = action
+      const users = [...state]
+      const i = users.findIndex(u => u.address === user.address)
+
+      return i === -1 ? [...users, user] : users.map(u => u.address === user.address ? user : u)
```

Comment:
Ternary happy 😬

Reply:
I think this is OK for now but I foresee turning the state into an object at some point so we can store other state, such as loading status, error codes etc.